src/mesh/Mesh.hpp

```diff
@@ -191,6 +191,13 @@ class Mesh {
   /// Computes the boundingBox for the vertices.
   void computeBoundingBox();
 
+  /// Computes if a quad is convex and returns and ordered set of vertices.
```

Comment:
```suggestion
  /// Computes if a quad is convex and returns an ordered set of vertices.
```
How is the information whether the quad is convex returned?

Reply:
I could add a check if a valid convex is true, and make the function a bool that returns true for a valid quad.
```
bool computeQuadConvexityFromPoints(std::array<int,4> &vertices) const;
```
If untrue, the check would fail and give an error about an invalid quad.